package.json

```diff
@@ -14,7 +14,8 @@
     "html-react-parser": "^5.1.10",
     "next": "13.5.6",
     "react": "^18",
-    "react-dom": "^18"
+    "react-dom": "^18",
+    "react-property": "^2.0.2"
```

Comment:
What is this being used for?

Reply:
resolved a conflict with main by installed react-property